k8s.io/OWNERS

```diff
@@ -1,5 +1,7 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
+options:
+  no_parent_owners: true
 approvers:
 - ixdy
```

Comment:
@ixdy are you still active here?

also happy to help here @nikhita

Reply:
no, I should probably be removed.

Reply:
I can do this as followup